cockatrice/src/remoteclient.cpp

```diff
@@ -150,6 +150,11 @@ void RemoteClient::loginResponse(const Response &response)
         if (resp.missing_features_size() > 0 && settingsCache->getNotifyAboutUpdates())
                 emit notifyUserAboutUpdate();
 
+        if (resp.server_features_size() > 0) {
+            for (int i = 0; i < resp.server_features_size(); ++i)
+                settingsCache->addServerFeatures(QString::fromStdString(resp.server_features(i)));
```

Comment:
I see a problem here: the ```serverFeatures``` variable in settingscache is never cleared. So, if i first connect to a server supporting a specific feature "A", then disconnect and connect to a second server not supporting the feature, it will still be listed in the available server features.
We should either clear the serverFeatures list when disconnecting, or just making the ```serverFeatures``` variable a property of the current active server.